src/avatar/__tests__/avatar.test.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { render } from '@testing-library/react';
+import React from 'react';
+import Avatar, { AvatarProps } from '../../../lib/components/avatar';
+import createWrapper from '../../../lib/components/test-utils/dom';
+import loadingDotsStyles from '../../../lib/components/avatar/loading-dots/styles.selectors.js';
+
+const defaultAvatarProps: AvatarProps = { ariaLabel: 'Avatar' };
+
+function renderAvatar(props: AvatarProps) {
+  const { container } = render(<Avatar {...props} />);
+
+  return createWrapper(container).findAvatar()!;
+}
+
+describe('Avatar component', () => {
+  describe('Basic', () => {
+    (['gen-ai', 'default'] as AvatarProps.Color[]).forEach(color => {
+      test(`renders ${color} color avatar`, () => {
```

Comment:
I don't think we need to test against every color. It is enough to add a single test that verifies the color is correctly asigned or even no unit test for it at all because such feature is much better to assure with VR/screenshot tests.

Reply:
That is correct, I believe I forgot to re-visit this part after updating the API from type/variant to color. I'll do the change.